<?xml version="1.0"?>
<robot name="curiosity_mars_rover" xmlns:xacro="http://www.ros.org/wiki/xacro">

    <!-- Inertia matrices -->
	<xacro:include filename="$(find mysim)/urdf/macros.xacro" />
	<!-- Robot constants and properties -->
	<xacro:include filename="$(find mysim)/urdf/curiosity_mars_rover_properties.xacro" />
	<!-- Gazebo aspects of the robot -->
	<xacro:include filename="$(find mysim)/urdf/curiosity_mars_rover.gazebo" />
	<!-- Chasis -->
	<xacro:include filename="$(find mysim)/urdf/chassis.xacro" />
	<!-- Wheel Groups -->
	<xacro:include filename="$(find mysim)/urdf/wheel.xacro" />
	<xacro:include filename="$(find mysim)/urdf/left_wheel_group.xacro" />
	<xacro:include filename="$(find mysim)/urdf/right_wheel_group.xacro" />
	<!-- Arm -->
	<xacro:include filename="$(find mysim)/urdf/arm.xacro" />
	<!-- Sensor Mast -->
	<xacro:include filename="$(find mysim)/urdf/sensor_mast.xacro" />

	<chassis_body/>
	<left_wheel_tree/>
	<right_wheel_tree/>

	<arm/>
	<sensor_mast/>


</robot>